src/test/java/edu/cornell/kfs/rass/batch/RassGrantDescriptionConverterTest.java

```diff
@@ -0,0 +1,71 @@
+package edu.cornell.kfs.rass.batch;
+
+import static org.junit.Assert.assertEquals;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.core.config.Configurator;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.kuali.kfs.coreservice.framework.parameter.ParameterService;
+import org.mockito.Mockito;
+
+public class RassGrantDescriptionConverterTest {
+    private RassGrantDescriptionConverter rassGrantDescriptionConverter;
+
+    @Before
+    public void setUp() throws Exception {
+        Configurator.setLevel(RassGrantDescriptionConverter.class.getName(), Level.DEBUG);
+        rassGrantDescriptionConverter = new RassGrantDescriptionConverter();
+        rassGrantDescriptionConverter.setParameterService(buildMockParameterService());
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        rassGrantDescriptionConverter = null;
+    }
+    
+    private ParameterService buildMockParameterService() {
+        ParameterService service = Mockito.mock(ParameterService.class);
+        //public String getSubParameterValueAsString(String namespaceCode, String componentCode, String parameterName, String subParameterName) {
+        
+        for (TestData data : TestData.values()) {
+            Mockito.when(service.getSubParameterValueAsString(Mockito.anyString(), Mockito.anyString(), Mockito.anyString(), Mockito.eq(data.searchValue))).
+                thenReturn(data.parameterServiceResults);
```

Comment:
I think the dot should be at the start of the second line instead of the end of the first line.

Reply:
OK, I will change